From 5f94677b36fc17546a18469eaaff6db28ac56f44 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:23 -0400
Subject: [tools] perf: Add missing event for perf sched record

Message-id: <1410181752-25631-199-git-send-email-jolsa@redhat.com>
Patchwork-id: 91969
O-Subject: [PATCH RHEL7.1 BZ1134356 198/307] perf tools: Add missing event for perf sched record.
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 7fff959783949b2f50454c49e325697073f48dc0
Author: Dongsheng <yangds.fnst@cn.fujitsu.com>
Date: Mon May 5 16:05:53 2014 +0900

description
===========
We should record and process sched:sched_wakeup_new event in
perf sched tool, but currently, there is the process function
for it, without recording it in record subcommand.

This patch add -e sched:sched_wakeup_new to perf sched record.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-sched.c b/tools/perf/builtin-sched.c
index d3fb0ed..7eae501 100644
--- a/tools/perf/builtin-sched.c
+++ b/tools/perf/builtin-sched.c
@@ -1635,6 +1635,7 @@ static int __cmd_record(int argc, const char **argv)
   "-e", "sched:sched_stat_runtime",
   "-e", "sched:sched_process_fork",
   "-e", "sched:sched_wakeup",
+  "-e", "sched:sched_wakeup_new",
   "-e", "sched:sched_migrate_task",
  };
 
-- 
1.7.1